src/log.c

```diff
@@ -124,7 +124,11 @@ static int init_logging_session(const Client_Config *c_config, const char *name,
     return 0;
 }
 
-#define LOG_FLUSH_LIMIT 1  /* limits calls to fflush to a max of one per LOG_FLUSH_LIMIT seconds */
+/* limits calls to fflush to a max of one per LOG_FLUSH_LIMIT seconds */
+#define LOG_FLUSH_LIMIT 1
+
+/* Maximum amount of data we can write to a log in a given session */
+#define LOG_MAX_SIZE ((1024 << 10) * 100)  // 100 MiB
```

Comment:
`1024 << 10` is `1 << 20` or `1024 * 1024` (which I see more commonly used when making kibibytes and mebibytes). What made you choose the first version?

Reply:
I'm following the pattern of:
`1024 == 1KiB`
`1024 << 10 == 1Mib`
`1024 << 20 == 1GiB`
`1024 << 30 == 1TiB`
etc.

So if we want `n` MiB we do `(1024 << 10) * n`

Reply:
SGTM.